base/settings/R/createMultisiteMultiSettings.r

```diff
@@ -73,12 +78,21 @@ createSitegroupMultiSettings <- function(
 #' @example examples/examples.MultiSite.MultiSettings.r
 createMultiSiteSettings <- function(templateSettings, siteIds) {
   templateSettings <- as.MultiSettings(templateSettings)
+
+  if (is.data.frame(siteIds)) {
+    ids <- siteIds$id
```

Comment:
We usually use `site_id` instead of `id` when extracting from the `site_info` list.

Reply:
Good to know, but this also needs to align with workflows that use `PEcAn.DB::query.site()`, which uses `id`

Reply:
Ok. Sounds like some discrepancies between pecan settings and SDA workflows.